apps/E2E/src/ButtonExperimental/specs/ButtonExperimental.spec.win.ts

```diff
@@ -37,7 +37,7 @@ describe('Experimental Button Accessibility Testing', async () => {
     await expect(await ButtonExperimentalPageObject.didAssertPopup()).toBeFalsy(ButtonExperimentalPageObject.ERRORMESSAGE_ASSERT);
   });
 
-  it('Experimental Button - Set accessibilityLabel', async () => {
+  it('Set "accessibilityLabel" prop. Validate "accessibilityLabel" value propagates to "Name" element attribute.', async () => {
```

Comment:
Not sure I like adding "propagates to "XXX" element attribute." as part of the test descriptions. I suppose it's helpful in letting the user know which native property it's looking for 🤔

Reply:
The "propagates to "XXX" element attribute" is more verbose, but is a lot more clear on what the test actually does as opposed to stating "is correct". I guess it's better to be over-descriptive than vague.

Reply:
I can agree with that.